.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,46 @@
+name: Snyk test Docker image
+on: 
+  push:
+    branches: [master, radix]
+  workflow_dispatch:
+
+jobs:
+  snyk_test:
+    name: Snyk test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Load openvds image
+      uses: "./.github/actions/load_openvds_image"
+
+    - name: Set docker image tag name for snyk testing
+      id: set_tag
+      run: |
+        tag=$(echo "snyk/${{ github.repository }}/${GITHUB_REF#refs/heads/}" \
```

Comment:
I think it could make sense to change
`snyk/${{ github.repository }}/${GITHUB_REF#refs/heads/}`
to something like
`${{ github.repository }}/docker-image-${ github.ref_name }` because Snyk uses the tag as name in the dashboard.

To me this would have the following advantages:
- It is ok to leave out the `snyk/` prefix since it does not add any useful information in the Snyk dashboard (we are on Snyk so we know that it is related to Snyk.
- This should ensure that the scanned images get the same prefix, i.e., `equinor/vds-slice`, in the Snyk dashboard as the other `vds-slice` related scans.
- Makes it more clear in the Snyk dashboard what the scan result is for (the Docker image).

Reply:
Good suggestion! 😇